ospo-book/content/en/03-chapter.md

```diff
@@ -59,6 +59,10 @@ The structure used in this book to represent these areas is shaped as a flower d
 
 #### Creating and Implementing an Open Source Strategy
 
+> If an organization's primary objective is profitability, customer satisfaction becomes a linchpin in the corporate strategy – a focal point for both CEOs and CFOs. For individuals in Open Source Program Offices, effectively communicating the open source strategy to C-level executives demands a keen understanding of the industry landscape and alignment with the key considerations of CEOs and CFOs. This alignment necessitates a clear comprehension of the overarching corporate strategy and identifying technologies within the open-source realm that can propel the organization toward its strategic objectives
+>
+> Victor Lu - [Strategy - End Game for FINOS Maturity Model](https://docs.google.com/presentation/d/1jJtR6-fvU-dCrGq_gTm4P1Awv90oCu4RClj1919970A/edit?usp=sharing)
```

Comment:
Is this a quote from Victor's presentation? I could not find the actual quote in the slides, maybe it was mentioned when the presentation we delivered? In this case we should link the video.

Reply:
That comes from a summary of the slides written by Victor himself 🙂 He shared this during one of the past calls